src/tito/tagger/main.py

```diff
@@ -132,10 +133,42 @@ def _tag_release(self):
         new_version = self._bump_version()
         self._check_tag_does_not_exist(self._get_new_tag(new_version))
         self._update_changelog(new_version)
+        # Update pkg managers used by the project (e.g. Cargo, Cabal)
+        # XXX: merge setup_py and pom_xml into this method ?
+        self._update_pkg_managers(new_version)
         self._update_setup_py(new_version)
         self._update_pom_xml(new_version)
         self._update_package_metadata(new_version)
 
+    def _update_pkg_managers(self, new_version):
+        section_name = 'remote_pkg_manager'
+        option_type = 'type'
+        option_file = 'config_file'
```

Comment:
Question for you, why drive this by config vs just detecting a Config.toml, and assuming that means we should try to update a version in it? (I believe this is what we do for setup.py) Is the config helpful or would it be safe to just automatically do it. (perhaps failing gracefully if there's no version in there for some reason)

If config is better, remote_pkg_manager feels ill-named, what is the remote referring to? Things like "pkg_metadata" are coming to mind if we were to apply this config to setup.py and pom.xml. (though we can't really as that would change automatic behaviour for users now, might be another indication we should think about making this happen automatically)